tests/chocolatey-tests/chocolatey.Tests.ps1

```diff
@@ -365,4 +365,29 @@ Describe "Ensuring Chocolatey is correctly installed" -Tag Environment, Chocolat
             $LASTEXITCODE | Should -Be 0
         }
     }
+
+    # This is skipped when not run in CI because it modifies the local system.
+    Context '.Net Registry is not set' -Skip:(-not $env:TEST_KITCHEN) {
+        BeforeAll {
+            $RegistryPath = 'SOFTWARE\Wow6432Node\Microsoft\NET Framework Setup\NDP\v4\Full\'
+            Set-RegistryKeyOwner -Key $RegistryPath
+            $OriginalRelease = Get-ItemPropertyValue -Path "HKLM:\$RegistryPath" -Name Release
+            Remove-ItemProperty -Path "HKLM:\$RegistryPath" -Name Release
+            $Output = Invoke-Choco help
+            Write-Host $Output.String
+            Get-ItemPropertyValue -Path "HKLM:\$RegistryPath" -Name Release | Out-String | Write-Host
+        }
+
+        AfterAll {
+            Set-ItemProperty -Path "HKLM:\$RegistryPath" -Name Release -Value $OriginalRelease

```

Comment:
Should the owner be set back to original? Since I'm not seeing code to do so. Although I think it probably would not matter?

Reply:
I'd love to make that be the case, but apparently it's not as easy as just asking it to put back `NT SERVICE\TrustedInstaller` 😂 

![2023-01-11_17-27-02](https://user-images.githubusercontent.com/30301021/211954128-12c02fc7-9c9a-4a0e-b994-2ff84159bddf.png)

But also: This test should only run in CI or a very specific configuration of the Vagrantfile, so it shouldn't affect a developer's system.

Reply:
Sounds good to me :+1: